x-pack/plugins/apm/server/routes/typings.ts

```diff
@@ -41,7 +42,7 @@ export interface APMRouteCreateOptions {
     >;
     body?: { accepts: Array<'application/json' | 'multipart/form-data'> };
     disableTelemetry?: boolean;
-  };
+  } & RouteConfigOptions<any>;
 }
```

Comment:
Is `any` necessary here?

Reply:
Yes, it's a required param. Ideally we'd pass the request method but that's a little tricky since that's part of the `endpoint` string. I _think_ it can be extracted with string literal types but I didn't bother tbh.